python/ray/data/tests/test_resource_manager.py

```diff
@@ -138,3 +141,55 @@ def test_calculating_usage(self):
         assert resource_manager.get_downstream_object_store_memory(o1) == 1700
         assert resource_manager.get_downstream_object_store_memory(o2) == 1700
         assert resource_manager.get_downstream_object_store_memory(o3) == 1000
+
+    def test_object_store_usage(self):
```

Comment:
should add `restore_data_context` for this test.

Reply:
Ah, good catch. 

Wonderiing why we don't just enable this by default for all tests?